app/lib/three_scale/policies/specification.rb

```diff
@@ -6,18 +6,43 @@ class Specification
       def initialize(doc)
         @errors = ActiveModel::Errors.new(self)
         @doc = doc
-        @validator = JSONValidator.new(doc)
       end
 
       attr_reader :errors, :doc
 
-      JSON_SCHEMA = {'$ref' => 'http://apicast.io/policy-v1/schema#'}.freeze
+      SCHEMAS_PATH = 'app/lib/three_scale/policies/schemas/'
+      POLICY_SCHEMAS_FILENAMES = %w[apicast-policy-v1.1.schema.json apicast-policy-v1.schema.json].freeze
+
+      POLICY_SCHEMAS = POLICY_SCHEMAS_FILENAMES.each_with_object({}) do |schema_filename, schemas|
+        policy_schema = JSON.parse(File.read(File.join(SCHEMAS_PATH, schema_filename)))
+        schema_id = policy_schema["$id"]
+        schemas[schema_id] = JSONSchemer.schema(policy_schema) if schema_id
+      end.freeze
```

Comment:
why not use `map` here? we can have duplicates or something?

Reply:
Because I have an array and I want a hash.

I could do `.map` returning `[schema_id, JSONSchemer.schema(policy_schema)]` and then call `.to_h` on the returned array of arrays, but I am not sure it would be any better :thinking: 

I think this way it is very clear what is the key and what is the value in `POLICY_SCHEMAS`.